src/handlers/dataProvidersHandlers.ts

```diff
@@ -83,6 +84,20 @@ const putDataProvider = (
 		);
 		return;
 	}
+	const { keycloakOrganizationId } = req.params;
```

Comment:
Should this be added to the OpenAPI spec?

Reply:
Of course, yes!